test/README.md

```diff
@@ -91,3 +91,62 @@ When a new version of Docker is released, you willneed to update the GCE test im
 To do this, change the Docker version in `run-integration-tests.sh` and push the change.
 Next build in CircleCI will detect that there is no template for this version of Docker and will first create the template before running tests.
 Subsequent builds will then simply re-use the template.
+
+## Running the tests on a cloud provider
+Running the tests against a remote provider instead of local vagrant VMs is a little more involved, and reproduces the environment used for CI. It is recommended that you use the build VM to coordinate the build phase.
+
+1. Create the build VM and log in
```

Comment:
To avoid possible confusions, maybe it's worth specifying from which dir to create the VM, i.e. either `$WEAVE_ROOT` or `$WEAVE_ROOT/test`?